polyfill/intersection-observer-test.html

```diff
@@ -0,0 +1,95 @@
+<!--
+Copyright 2016 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>IntersectionObserver Tests</title>
+
+  <!-- Dependencies -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mocha/2.5.3/mocha.min.css">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/mocha/2.5.3/mocha.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/expect.js/0.2.0/expect.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/sinon.js/1.15.4/sinon.min.js"></script>
+
+  <!-- Polyfills for IE8 -->
+  <script>
+  if (!('getComputedStyle' in window && 'bind' in Function.prototype)) {
+    document.write(
+        '<script src="https://cdnjs.cloudflare.com/ajax/libs/' +
+        'es5-shim/4.5.8/es5-shim.min.js"><\/script>' +
+        '<script src="https://cdn.rawgit.com/Financial-Times/polyfill-' +
+        'service/master/polyfills/getComputedStyle/polyfill.js"><\/script>');
+  }
+  </script>
+
+  <!-- Styles required for tests -->
+  <style>
+  #root {
+    position: relative;
+    width: 400px;
+    height: 200px;
+    background: #eee
+  }
+  #grand-parent {
+    position: relative;
+    width: 200px;
+    height: 200px;
+  }
+  #parent {
+    position: absolute;
+    top: 0px;
+    left: 200px;
+    overflow: hidden;
+    width: 200px;
+    height: 200px;
+    background: #ddd;
+  }
+  #target1, #target2, #target3, #target4 {
+    position: absolute;
+    top: 0px;
+    left: 0px;
+    width: 20px;
+    height: 20px;
+    transform: translateX(0px) translateY(0px);
+    transition: transform .5s;
+    background: #f00;
+  }
+  </style>
```

Comment:
This makes it quite hard for tests to be imported into a larger set.  The previous test suite injected the necessary styles as needed onto the test elements.  This would help the polyfill service.  Meanwhile I'll use a hack to pull these in.